doc/releases/changelog-dev.md

```diff
@@ -83,6 +83,13 @@
 
 <h4>Mid-circuit measurements and dynamic circuits</h4>
 
+* The `default.qubit` device implements a depth-first tree-traversal algorithm to
+  accelerate native mid-circuit measurement execution. The new implementation
+  supports classical control, collecting statistics, and post-selection, along
+  with all measurements enabled with `qml.defer_measurements` or
```

Comment:
`defer_measurements` is not limited to `var`, `expval`, `probs`, `sample`, and `counts`. That is only the case if the MP is using a `MeasurementValue`. If that is the case with tree traversal then I'm happy with this entry, but if not, it might be worth specifying which measurements are supported.

Reply:
I hadn't thought about that. Better just compare with `qml.dynamic_one_shot` then? It is similarly limited because it has custom logic to combine each measurement.